content/webapp/components/IIIFViewer/MainViewer.tsx

```diff
@@ -211,7 +211,7 @@ function getPositionData({
   return highlightsPositioningData || [];
 }
 const ItemRenderer = memo(({ style, index, data }: ItemRendererProps) => {
-  const { scrollVelocity, canvases, externalAccessService, accessToken } = data;
+  const { scrollVelocity, canvases, externalAccessService } = data;
```

Comment:
Just flagged as declared but unused so removed it

Reply:
I'm using it in my branch, but can add it back in